src/openms/include/OpenMS/ANALYSIS/ID/MetaboliteSpectralMatching.h

```diff
@@ -54,140 +54,132 @@ namespace OpenMS
+    String getCommonName() const;
+    void setCommonName(const String&);
 
-      String getSumFormula() const;
-      void setSumFormula(const String&);
+    String getSumFormula() const;
+    void setSumFormula(const String&);
 
-      String getInchiString() const;
-      void setInchiString(const String&);
+    String getInchiString() const;
+    void setInchiString(const String&);
 
-      String getSMILESString() const;
-      void setSMILESString(const String&);
+    String getSMILESString() const;
+    void setSMILESString(const String&);
 
-      String getPrecursorAdduct() const;
-      void setPrecursorAdduct(const String&);
+    String getPrecursorAdduct() const;
+    void setPrecursorAdduct(const String&);
 
 
   private:
-      double observed_precursor_mass_;
-      double observed_precursor_rt_;
-      double found_precursor_mass_;
-      Int found_precursor_charge_;
-      double matching_score_;
-      Size observed_spectrum_idx_;
-      Size matching_spectrum_idx_;
-
-      // further meta information
-      String primary_id_;
-      String secondary_id_;
-      String common_name_;
-      String sum_formula_;
-      String inchi_string_;
-      String smiles_string_;
-      String precursor_adduct_;
+    double observed_precursor_mass_;
+    double observed_precursor_rt_;
+    double found_precursor_mass_;
+    Int found_precursor_charge_;
+    double matching_score_;
+    Size observed_spectrum_idx_;
+    Size matching_spectrum_idx_;
+
+    // further meta information
+    String primary_id_;
+    String secondary_id_;
+    String common_name_;
+    String sum_formula_;
+    String inchi_string_;
+    String smiles_string_;
+    String precursor_adduct_;
 
   };
 
   struct OPENMS_DLLAPI SpectralMatchScoreComparator
   {
-      bool operator() (const SpectralMatch& a, const SpectralMatch& b)
-      {
-          return a.getMatchingScore() > b.getMatchingScore();
-      }
+    bool operator() (const SpectralMatch& a, const SpectralMatch& b)
+    {
+      return a.getMatchingScore() > b.getMatchingScore();
+    }
 
   } SpectralMatchScoreGreater;
 
-
-
-
   class OPENMS_DLLAPI MetaboliteSpectralMatching :
   public DefaultParamHandler,
   public ProgressLogger
   {
   public:
-      /// Default constructor
-      MetaboliteSpectralMatching();
-
-      /// Default destructor
-      virtual ~MetaboliteSpectralMatching();
+    /// Default constructor
+    MetaboliteSpectralMatching();
 
-      /// hyperscore computation
-      double computeHyperScore(MSSpectrum<Peak1D>, MSSpectrum<Peak1D>, const double&, const double&);
+    /// Default destructor
+    virtual ~MetaboliteSpectralMatching();
 
-      /// main method of MetaboliteSpectralMatching
-      void run(PeakMap&, MzTab&);
+    /// hyperscore computation
+    double computeHyperScore(MSSpectrum<Peak1D>, MSSpectrum<Peak1D>, const double&, const double&);
 
+    /// main method of MetaboliteSpectralMatching
+    void run(PeakMap &, PeakMap &, MzTab &);
```

Comment:
I always wondered if it would be possible to "tag" classes and members with pxd code to allow automatic extraction of a pxd file (at least for most classes). Would simplify refactoring and is easier spotted in the review.